src/alto-ui/Form/Typeahead/Typeahead.js

```diff
@@ -41,15 +41,21 @@ const renderFields = (search, itemToString, fields) => (item, ...args) => {
   return fields(item, itemToStringFromated, ...args);
 };
 
-function typeaheadStateReducer(state, changes) {
+function typeaheadStateReducer(state, changes, documentRef, dropdownRef) {
   switch (changes.type) {
     // ON BLUR (on click outside)
-    case Downshift.stateChangeTypes.mouseUp:
+    case Downshift.stateChangeTypes.mouseUp: {
+      // We have to check if user clicks on scrollbar, because on IE it triggers onBlur event
+      // If yes, we don't want to hide dropdown list
+      const wasScrollBarClicked = documentRef.current.activeElement.isSameNode(dropdownRef.current);
```

Comment:
It can be destructurized:
```
const {
  current: {
    activeElement: {
      isSameNode,
    },
  },
  current: currentRef,
} = documentRef;
const wasScrollBarClicked = isSameNode(currentRef);
```

Reply:
We cannot use `isSameNode` func without context. But rest was destructurized